pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["flit_core >=2,<3"]
+requires = ["flit_core >=2"]
```

```suggestion
requires = ["flit_core >=4"]
```
As libraries can have breaking changes on major versions, it's typically good practice to have an upper pin. 

That said - it looks like v3.x of `flit_core` works fine, so we could consider extending the range to `<4`.